docs/en/docs/tutorial/response-model.md

```diff
@@ -179,7 +179,7 @@ They take a `set` of `str` with the name of the attributes to include (omitting
 This can be used as a quick shortcut if you have only one Pydantic model and want to remove some data from the output.
 
 !!! tip
-    But it is still recommended to use the ideas above, using multiple classes, instead of these parameters.
+    But it is still recommended using the ideas above, using multiple classes, instead of these parameters.
```

Comment:
I think the old version was better.
Because here, it's not the case of "using X == using Y".
Rather it's "to use X, which implies using Y (within)".